Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -140,7 +130,8 @@ def render_PUT(self, request):
         try:
             channel_id = self.session.create_channel(parameters['name'][0], description, mode)
         except DuplicateChannelNameError as ex:
-            return MyChannelBaseEndpoint.return_500(self, request, ex)
+            self._logger.exception(ex)
```

Comment:
No, because this is for handling errors that are specific to the use case to differentiate from crashes.
On a higher level unhandled errors are caught and logged.